SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -59,201 +56,127 @@ void CDriver::PythonInterface_Preprocessing(CConfig **config, CGeometry ****geom
       }
     }
   }
-  /*--- Initialize some variables used for external communications trough the Py wrapper. ---*/
-  PyWrapVarCoord[0] = 0.0;
-  PyWrapVarCoord[1] = 0.0;
-  PyWrapVarCoord[2] = 0.0;
-  PyWrapNodalForce[0] = 0.0;
-  PyWrapNodalForce[1] = 0.0;
-  PyWrapNodalForce[2] = 0.0;
-  PyWrapNodalForceDensity[0] = 0.0;
-  PyWrapNodalForceDensity[1] = 0.0;
-  PyWrapNodalForceDensity[2] = 0.0;
-  PyWrapNodalHeatFlux[0] = 0.0;
-  PyWrapNodalHeatFlux[1] = 0.0;
-  PyWrapNodalHeatFlux[2] = 0.0;
 
 }
 
-passivedouble CDriver::Get_Drag() {
+/////////////////////////////////////////////////////////////////////////////
+/* Functions related to the global performance indices (Lift, Drag, ecc..) */
+/////////////////////////////////////////////////////////////////////////////
```

Comment:
Good to see those temporaries going away and the different methods organized :+1: